roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -50,14 +49,12 @@
 
 # Inject ownerReferences
 - name: Inject owner references for KubevirtCommonTemplatesBundle
-  include_role: 
-    name: ClaimOwnership
-  vars:
-    object: "{{ item }}"
-    objectKind: "templates"
-    owner: "{{ cr_info }}"
-  when: "{{ old_cr_exists==true }}"
-  with_items: "{{ old_cr_templates }}" # Templates
+  k8s:
```

Comment:
The `ClaimOwnership` role can be removed as it is not being used anymore

Reply:
And what about here https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/roles/KubevirtNodeLabeller/tasks/main.yml#L30?

Reply:
Ah, I was too focused on common-templates.
Maybe you can replace it with your new filter?